services/web/client/source/class/qxapp/auth/ui/RegistrationPage.js

```diff
@@ -0,0 +1,120 @@
+
+/**
+ *
+ *  TODO: add a check to prevent bots to register users
+*/
+qx.Class.define("qxapp.auth.ui.RegistrationPage", {
+  extend: qxapp.auth.core.BaseAuthPage,
+
+
+  /*
+  *****************************************************************************
+     MEMBERS
+  *****************************************************************************
+  */
+
+  members: {
+
+    // overrides base
+    _buildPage: function() {
+      let manager = new qx.ui.form.validation.Manager();
+
+      this._addTitleHeader(this.tr("Registration"));
+
+      // email, pass1 == pass2
+      let email = new qx.ui.form.TextField();
+      email.setRequired(true);
+      email.setPlaceholder(this.tr("Introduce your email"));
```

Comment:
```javascript
new qx.ui.form.TextField().set({
  required: true,
  placeholder: this.tr("Introduce your email")
});

Reply:
@oetiker I also like more this way but ... besides aesthetics, is there an additional advantage to set properties this way ? (e.g. atomicity  )